src/app/window-handler.ts

```diff
@@ -258,7 +280,7 @@ export class WindowHandler {
                 return this.destroyAllWindow();
             }
 
-            if (this.config.minimizeOnClose) {
+            if (config.getConfigFields([ 'minimizeOnClose' ]).minimizeOnClose) {
```

Comment:
why do we need this change?

Reply:
`config.getConfigFields` always returns the latest from `config` file whereas `this.config` is only for initial read